src/test/java/edu/tamu/cap/controller/SchemaControllerTest.java

```diff
@@ -42,7 +45,7 @@
 
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
-@RunWith(SpringRunner.class)
+@ExtendWith(SpringExtension.class)
```

Comment:
`@SpringBootTest` already includes this, please remove `ExtendWith(SpringExtension.class)`.